src/main/java/bio/terra/cli/command/Main.java

```diff
@@ -104,17 +108,29 @@ public static int runCommand(String... args) {
    * @param args from stdin
    */
   public static void main(String... args) {
+    argList = Arrays.asList(args);
     // run the command
     int exitCode = runCommand(args);
 
     // set the exit code and terminate the process
     System.exit(exitCode);
+    resetArgList();
```

Comment:
Will this line ever be reached given that it's after a `System.exit` call? Regardless, I think no need to reset this property because the `Main` class should run exactly once per command.